src/main/java/uk/gov/hmcts/reform/pip/account/management/service/MediaLegalApplicationService.java

```diff
@@ -106,6 +114,8 @@ public MediaAndLegalApplication updateApplication(UUID id, MediaLegalApplication
         MediaAndLegalApplication applicationToUpdate = mediaLegalApplicationRepository.findById(id).orElseThrow(() ->
             new NotFoundException(String.format("Application with id %s could not be found", id)));
 
+        log.info(writeLog(UserActions.UPDATE_MEDIA_APPLICATION, applicationToUpdate.getEmail()));
```

Comment:
do we need to track which admin did the task?

Reply:
unfortunately the admin user doesn't come through to the backend atm :( However I have added logging into this for the frontend, so we'll get the admin log captured there when they update